ci-integration/virtualization/requirements_test.txt

```diff
@@ -4,4 +4,7 @@ pytest-timeout
 mock
 websocket-client
 numpy>1.13<2
```

Comment:
These actually should be in the individual agent's requirements.txt files and they are automatically installed when instlal-agent.py is run.

Reply:
To be revisited later.